### PR TITLE
pycln-fix

### PR DESCRIPTION
**Pull request by rodolfocarobene**
Adding `--all` parameter to `pycln`. 
See that I had to add an exception for pint-pandas in data.py, one of the unused/used imports...

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

File: src/qibocal/data.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 import numpy as np
 import pandas as pd
-import pint_pandas
+import pint_pandas  # nopycln: import
```
Comment (AleCandido) [inline src/qibocal/data.py:8]: ?

Comment (rodolfocarobene) [reply src/qibocal/data.py:8]: If I remove that import, several tests fail (I suppose every tests that uses this file).
I am not sure why, but I think that it's implicitly using it with
https://github.com/qiboteam/qibocal/blob/0b72989e53f66be52d0ac21c37e2357025880082/src/qibocal/data.py#L76-L82

Comment (AleCandido) [reply src/qibocal/data.py:8]: Here our first side effect. Most likely `pint_pandas` is patching `pandas` (i.e. replacing some content of the module) upon import.

Luckily, we have #372 :)

Comment (andrea-pasquale) [reply src/qibocal/data.py:8]: Indeed this is just how pint works for some reason...

**Review by andrea-pasquale — APPROVED** [review]
Thanks @rodolfocarobene

File: src/qibocal/data.py (hunk 1 above)
Comment (AleCandido) [reply src/qibocal/data.py:8]: It wants to be a *drop-in* replacement of Pandas, that's the reason. But, yeah, I don't like it so much...

**Pull request merged**